lib/experimental/fonts-library/class-wp-fonts-library.php

```diff
@@ -0,0 +1,618 @@
+<?php
+include(ABSPATH . "wp-admin/includes/admin.php");
+
+/**
+ * Font library class.
+ *
+ * @package gutenberg
+ * @since x.x.x
+ * 
+ */
+class WP_Fonts_Library extends WP_REST_Controller {
```

Comment:
Classes that extend the WP_REST_Controller class typically have the suffix "_Controller" in their name as well.
```suggestion
class WP_Fonts_Library_Controller extends WP_REST_Controller {
```